contracts/SL2RD.sol

```diff
@@ -179,6 +179,11 @@ contract SL2RD is
         return _communitySplitsBasisPoints.value;
     }
 
+    /// @notice Returns the next available community slot id.
+    function nextAvailableCommunitySlot() public view returns (uint256) {
```

Comment:
@DeusExM4ch1n4 Can we name this `countAllocatedCommunitySlots` or similar? I think count semantics will easier to reason about in the UI.